src/contracts/SavingCircles.sol

```diff
@@ -155,8 +174,11 @@ contract SavingCircles is ISavingCircles, ReentrancyGuard, OwnableUpgradeable {
 
     if (_isDecommissioned(_circle)) revert NotCommissioned();
 
-    _balances = new uint256[](_circle.members.length);
-    for (uint256 i = 0; i < _circle.members.length; i++) {
+    //Caching the circle members length to avoid multiple lookups
```

Comment:
same comment about this comment not being necessary. remove or add space after //